### ISSUE TITLE
VLP是否对LM进行了微调

### ISSUE THREAD

**Issue by Buganisher** [Thinklab-SJTU/Awesome-LLM4AD#8]
您好，在VLP原文中LM是冻结的，只对用于转化LM输出的适配器层进行训练，我认为将VLP归为微调预训练模型仍待讨论

**Comment by jayyoung0802** [comment]
Good point! We have reorganized the writing and will publish an updated version of the survey paper later this month.

**Comment by Buganisher** [comment]
> Good point! We have reorganized the writing and will publish an updated version of the survey paper later this month.

looking forward to your update
